src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -97,7 +97,7 @@ public function getResource()
         return $this->connection;
     }
 
-    public function setOption(string $name, $value)
+    public function setOption(string $name, float | int | string $value)
```

Comment:
according to https://www.php.net/manual/en/function.ldap-set-option.php, this should be `array|string|int|bool`, isn't it?
(please ignore fabbot on this one and don't add the spaces around `|`

Reply:
you're right, done